common/src/main/scala/services/teamwork/SquadService.scala

```diff
@@ -2230,8 +2230,8 @@ class SquadService extends Actor {
     val leadPosition = squad.Membership(0)
     leadPosition.Name = name
     leadPosition.CharId = player.CharId
-    leadPosition.Health = player.Health
-    leadPosition.Armor = player.Armor
+    leadPosition.Health = StatConverter.Health(player.Health, player.MaxHealth, min=1, max=64)
+    leadPosition.Armor = StatConverter.Health(player.Armor, player.MaxArmor, min=1, max=64)
```

Comment:
From a quick glance at `StatConverter` it will likely fail on cloakers. They have zero armour and it has no check for `max > 0`, leading to a division by zero on the final line

Reply:
That's what the first case is for - `if(health < 1) 0`.

And I know it needs renaming.

Reply:
For some reason it hadn't occured to me than if "MaxArmour" is 0, "Armour" will also be.

As you were. 😑